### PR TITLE
ORC-977: Update webpages and TestVectorOrcFile.java to be more neutral

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
### What changes were proposed in this pull request?

This PR aims to make our codebase and websites a little more neutral.
In addition, it fixes some typos.

### Why are the changes needed?

Although we are already careful, this will improve our awareness.

### How was this patch tested?

N/A

### CODE REVIEW

File: site/_docs/building.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ dockerUrl: https://github.com/apache/orc/blob/main/docker
 
 The C++ library is supported on the following operating systems:
 
-* CentOS 6 or 7
+* CentOS 7 or 8
```
Comment (dongjoon-hyun) [inline site/_docs/building.md:12]: This is a leftover which we missed.

File: site/_posts/2021-02-08-panagiotis-pmc.md [hunk 2]
```diff
@@ -10,5 +10,5 @@ On behalf of the Apache ORC Project Management Committee (PMC), it gives
 me great pleasure to announce that Panagiotis Garefalakis has joined the PMC. Panagiotis
 has radically improved the integration between Hive and ORC.
 
-Please join me in welcoming Dongjoon to the ORC PMC!
```
Comment (dongjoon-hyun) [inline site/_posts/2021-02-08-panagiotis-pmc.md:13]: This was a typo made by me.

**Comment by dongjoon-hyun** [comment]
cc @pgaref , @wgtmac , @williamhyun

**Review by omalley — APPROVED** [review]
LGTM +1

**Comment by dongjoon-hyun** [comment]
Thank you, @omalley !

**Comment by williamhyun** [comment]
Looks good to me! Thank you.

**Pull request merged**